### ISSUE TITLE
await Linking.getInitialURL(); giving wrong deep link

### ISSUE THREAD

**Issue by imdeepakchahar** [expo/expo#9255]
Dependencies
"expo": "~37.0.3",

Build Apk with
expo build:android

My code:
let initialUrl = await Linking.getInitialURL();

Problem is: initialUrl returning value https://expo.io:443/@imchahardeepak/greenapple in build apk
and when I open this URL then my project link opens.. not the app, so what I do?

By browsing the link which is given by await Linking.getInitialURL(); .. app should be open but my expo project URL is opening

**Comment by brentvatne** [comment]
you need to add a `scheme` to your app.json, otherwise there is no way to deep link into it

**Comment by imdeepakchahar** [comment]
@brentvatne thanks for your answer in  #9255.

And one more thing...
In 1st time, nobody has given answer, there was just one answer that use the format of asking the issue, and the issue was closed from your side without answering.

2nd time there was one line about schema no detailed answer, and you edited it later, that's why I didn't get notification of that.

So I asked 3rd time. and it's a bad answer to say if you post it again I will block. Because if I got answer 1st Time then why should I need to ask again.

BTW Thanks for your answer it helped me a lot.

and  [#9255 (comment)](https://github.com/expo/expo/issues/9255#issuecomment-659664743)
